### PR TITLE
Update readme with template vars and goreleaser fixes

### PR DESCRIPTION
**Pull request by nikkictl**
Signed-off-by: Nikki Attea <nikki@sensu.io>

### CODE REVIEW

File: .goreleaser.yml [hunk 1]
```diff
@@ -22,12 +22,13 @@ builds:
       - 7
     targets:
       - darwin_amd64
+      - darwin_386
```
Comment (palourde) [inline .goreleaser.yml]: Do we really want to support 386 macOS binaries, considering Catalina no longer supports 32-bit apps? Maybe we should just remove it from `.bonsai.yml` instead

Comment (nikkictl) [reply .goreleaser.yml]: Yeah one or the other needs to happen because bonsai would fail as is. I can remove it from bonsai.yml.

Comment (palourde) [reply .goreleaser.yml]: @amdprophet WDYT?

**Review by palourde — APPROVED** [review]
👌

**Review by palourde — COMMENTED** [review]
:shipit:

**Pull request merged**